CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ This file is used to list changes made in each version of the haproxy cookbook.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+
+- testing for haproxy 2.2
+
+### Removed
+
+- testing for haproxy 1.9
```

Comment:
Should 1.8 also be removed?

Reply:
It's LTS through Q4 of 2022, so i figured I'd keep in the mix.